src/transformers/utils/fx.py

```diff
@@ -926,6 +1099,9 @@ def trace(
             complete_concrete_args_with_inputs_not_in_dummy_inputs (`bool`, *optional*, defaults to `True`):
                 If `True`, and `dummy_inputs` is specified, every argument that `root` can take that is not in
                 `dummy_inputs` and not in `concrete_args` will be added to `concrete_args`, otherwise does nothing.
+            trace_non_torch_object (`bool`, *optional*, defaults to `False`):
```

Comment:
I think this should be a `HFTracer` class attribute to match what is being done in PyTorch: https://github.com/pytorch/pytorch/blob/master/torch/fx/proxy.py#L26

Reply:
I'm not sure what's the benefit of having it a class attribute?